docs/testing/sideload-office-add-ins-for-testing.md

```diff
@@ -1,57 +1,62 @@
+1. In the command line starting at the root directory of your project, run the following command. Replace "{url}" with the URL that you copied.
 
     [!INCLUDE [npm start:web command syntax](../includes/start-web-sideload-instructions.md)]
 
 1. The first time you use this method to sideload an add-in on the web, you'll see a dialog asking you to enable developer mode. Select the checkbox for **Enable Developer Mode now** and select **OK**.
 
 1. You will see a second dialog box, asking if you wish to register an Office Add-in manifest from your computer. You should select **Yes**.
 
-1. Your add-in is installed. If it is an add-in command, it should appear on either the ribbon or the context menu. If it is a task pane add-in, the task pane should appear.
+1. Your add-in is installed. If it has an add-in command, it should appear on either the ribbon or the context menu. If it's a task pane add-in without any add-in commands, the task pane should appear.
+
+## Sideload an add-in on the web when using Visual Studio
 
-## Sideload an Office Add-in in Office on the web manually
+If you're using Visual Studio to develop your add-in, pressing F5 will automatically open an Office document in *desktop* Office, create a blank document, and sideload the add-in. When you want to sideload to *Office on the web*, the process to sideload is similar to manual sideloading to the web. The only difference is that you must update the value of the **SourceURL** element, and possibly other elements, in your manifest to include the full URL where the add-in is deployed.
+
+1. In Visual Studio, show the **Properties** window by choosing **View** > **Properties Window**.
+1. In the **Solution Explorer**, select the web project. This will display properties for the project in the **Properties** window.
+1. In the Properties window, copy the **SSL URL**.
+1. In the add-in project, open the manifest XML file. Be sure you are editing the source XML. For some project types, Visual Studio will open a visual view of the XML which won't work for the next step.
+1. Search and replace all instances of **~remoteAppUrl/** with the SSL URL you just copied. You'll see several replacements depending on the project type, and the new URLs will appear similar to `https://localhost:44300/Home.html`.
+1. Save the XML file.
+1. Open the context menu of the web project (for example, by right clicking on it) then choose **Debug** > **Start new instance**. This runs the web project without launching Office.
+1. From Office on the web, sideload the add-in using steps previously described in [Manually sideload an add-in to Office on the web](#manually-sideload-an-add-in-to-office-on-the-web).
+
+## Manually sideload an add-in to Office on the web
 
 This method doesn't use the command line and can be accomplished using commands only within the host application (such as Excel).
 
```

Comment:
Consider splitting out the first step so it's:
```
1. Open [Office on the web](https://office.com/). Open a document in **Excel**, **OneNote**, **PowerPoint**, or **Word**.
1. On the **Insert** tab on the ribbon in the **Add-ins** section, choose **Office Add-ins**.
```

Reply:
This suggestion, in effect, moves the first step of the "Manually ..." section to the end of the "... Visual Studio" section. The problem is that the "Manually ..." section is linked to from other articles and it has to have that first step in it.